src/current/v23.1/recommended-production-settings.md

```diff
@@ -577,65 +582,71 @@ For example, for a node with 3 stores, we would set the hard limit to at least 3
     session    required   pam_limits.so
     ~~~
 
-1.  Edit `/etc/security/limits.conf` and append the following lines to the file:
+1.  Set a limit for the number of open file descriptors. The specific limit you set depends on your workload and the hardware and configuration of your nodes.
```

Comment:
The changes to this file look much larger than they actually are. This content was refactored for 23.2+ at some point and 23.1 didn't get that refactor. This brings the 23.1 section back in line with 23.2+.